src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/JsonConverterContractResolver.cs

```diff
@@ -57,10 +61,30 @@ public class JsonConverterContractResolver : DefaultContractResolver
             { typeof(MeshRenderer), new MeshRendererJsonConverter() },
             { typeof(SkinnedMeshRenderer), new SkinnedMeshRendererJsonConverter() },
             { typeof(NavMeshAgent), new NavMeshAgentJsonConverter() },
-            { typeof(RGGameMetadataJsonConverter), new RGGameMetadataJsonConverter() }
 
-        };
+            { typeof(BehaviourActionData), new BehaviourActionDataJsonConverter() },
+            { typeof(BotAction), new BotActionJsonConverter() },
+            { typeof(CVImageBinaryData), new CVImageBinaryDataJsonConverter() },
```

Comment:
What does this block of code do? Is this a bug fix given a lot these are not related to copying and we are just adding them?